Assignment6/Array.cs

```diff
@@ -1,10 +1,84 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 
 namespace Assignment6
 {
-    public class Array<T>
+    public class Array<T> : System.Collections.Generic.ICollection<T>
     {
-        
+        private Dictionary<int, T> Data { get; set; }
+        public int Size { get; private set; }

```

Comment:
```suggestion
        public int Capacity { get; private set; }
```

He did specifically say to make a Capacity property in the assignment. Not sure the naming would matter, but I'd change it.